lib/vscode/build/lib/node.js

```diff
@@ -8,8 +8,8 @@ const path = require("path");
 const fs = require("fs");
 const root = path.dirname(path.dirname(__dirname));
 const yarnrcPath = path.join(root, 'remote', '.yarnrc');
-// NOTE@coder: Fix version due to .yarnrc removal.
-const version = process.versions.node;
+const yarnrc = fs.readFileSync(yarnrcPath, 'utf8');
```

Comment:
AFAIK this is no longer the case

Reply:
I was talking to @code-asher about this. He said it may have been fixed but remembers weird edge cases related to this.